src/main/java/org/commonjava/indy/service/archive/model/dto/HistoricalContentDTO.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (C) 2011-2021 Red Hat, Inc. (https://github.com/Commonjava/indy)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.commonjava.indy.service.archive.model.dto;
+
+public class HistoricalContentDTO
+{
+    private String buildConfigId;
+
+    private String trackId;
```

Comment:
I think the trackingId(buildId) is not needed anymore for this model per Petr's suggestion. Only buildCfgId is needed. So that means we don't need the tracking record from indy anymore in archive service.

Reply:
I missed this comment.
@ligangty , I still think we need the trackingId, I use it to name the content file and archive file. Meanwhile, we need trackingId to send the tracking report and seal records in main indy based on it, which means in the sidecar side, we need trackingId+ buildConfigId. So in the archive side, although we could do generate/retrieve/delete only based on buildConfigId, there should be some unexpected cases(archive service is hung or interrupted) which probably make the content/archive unaligned(not latest)?

If so, we could incorporate the trackingId in the archive service, and do some audit based on trackingId before sidecar build, don't just trust the buildConfigId, it should be more reliable.

I think it should not be difficult for PNC to provide trackingId, and I KNOW based on scoping/discussions we agree the buildConfigId is required, but from the preview of implementation, that could be a plus.

Please confirm on this, thank you!

Reply:
@ligangty I might forget something.
Sidecar will generate new trackingId based on archive, and send report. if so, sidecar should not have some inputs like old trackingId, just buildConfigId.
If it is yes, ignore my comments above, we could just use buildConfigId in archive service!

Reply:
@ligangty and for build trackingId version audit, it could be a plus in future.